### PR TITLE
wip: mvp

### PR DESCRIPTION
**Pull request by PECHIVKO**

### CODE REVIEW

File: project/repository/postgres/project.go [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer tx.Rollback()
+
+	model := toProject(p)
+
+	var projectID int
+
+	err = tx.QueryRowContext(ctx, insertProjectQuery, model.Name, model.Description).Scan(&projectID)
```
Comment (PECHIVKO) [inline project/repository/postgres/project.go:53]: туть

File: column/delivery/http/handler.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+	if err != nil {
+		log.Println(err)
+		respondWithError(w, http.StatusInternalServerError, err.Error())
+	} else {
+		respondwithJSON(w, http.StatusOK, map[string]string{"message": "Column successfully updated"})
+	}
+}
+
+func (h *Handler) Move(w http.ResponseWriter, r *http.Request) {
+	id, _ := strconv.Atoi(chi.URLParam(r, "id"))
```
Comment (eugeneradionov) [inline column/delivery/http/handler.go:70]: do not ignore errors

File: comment/delivery/http/handler.go [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+func (h *Handler) Update(w http.ResponseWriter, r *http.Request) {
+	input := new(updateNameInput)
+	err := json.NewDecoder(r.Body).Decode(&input)
+	if err != nil {
+		log.Println(err)
+		respondWithError(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+	input.CommentID, _ = strconv.Atoi(chi.URLParam(r, "id"))
```
Comment (eugeneradionov) [inline comment/delivery/http/handler.go:58]: do not ignore errors

File: comment/repository/postgres/comment.go [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+	"log"
+	"time"
+
+	"github.com/PECHIVKO/task-manager/models"
+)
+
+const (
+	insertCommentQuery = `insert into comments (task_id, comment)
+								  values ($1, $2);`
+	// checkForTaskExists = `select exists
```
Comment (eugeneradionov) [inline comment/repository/postgres/comment.go:16]: remove or uncomment

File: config/config.go [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+	"gopkg.in/go-playground/validator.v9"
+	"gopkg.in/yaml.v2"
+)
+
+type Config struct {
+	Database Database `yaml:"database" validate:"required"`
+}
+
+type Database struct {
+	//	MigrationsSource string `yaml:"migrations_source" validate:"required"`
```
Comment (eugeneradionov) [inline config/config.go:16]: remove or uncomment

**Review by eugeneradionov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**